src/components/Footer/Footer.js

```diff
@@ -1,17 +1,48 @@
 import React from 'react';
 import styled from 'styled-components';
-
 import { rem } from 'polished';
 
-const FooterText = styled.p`
+import { fonts } from 'styles/base';
+import { FormattedMessage } from 'gatsby-plugin-intl';
+
+const FooterMainText = styled.p`
   margin: ${rem(8)} 0;
   text-align: center;
 `;
 
-const Footer = () => (
-  <div>
-    <FooterText>Karol Zieniewicz &reg; 2020</FooterText>
-  </div>
-);
+const FooterSideText = styled.p`
+  margin: ${rem(4)} 0;
+  font-size: ${fonts.sizes.xs};
+  text-align: center;
+`;
+
+const Footer = () => {
+  const currentYear = new Date().getFullYear();
+  return (
```

Comment:
empty line would be nice. Also - you could use `new Date()` directly in the template to avoid `return` keyword